src/components/templates/Textarea/index.jsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import styled from '@emotion/styled'
+
+const textArea = ({style, className, onChange, value, name , placeholder})=> {
+    return (
+        <TextArea placeholder={placeholder} style={style} className={className} onChange={onChange} name={name} >
+        </TextArea>
```

Comment:
```suggestion
        <TextArea placeholder={placeholder} style={style} className={className} onChange={onChange} name={name} />
```
싱글 브라켓은 하나로 적어주시는 게 어떨까요? 😊